snforge_std/src/cheatcodes.cairo

```diff
@@ -179,3 +215,55 @@ fn get_class_hash(contract_address: ContractAddress) -> ClassHash {
     let buf = cheatcode::<'get_class_hash'>(array![contract_address_felt].span());
     (*buf[0]).try_into().expect('Invalid class hash value')
 }
+
+fn start_spoof(contract_address: ContractAddress, tx_info_mock: TxInfoMock) {
+    let contract_address_felt: felt252 = contract_address.into();
+
+    let TxInfoMock{
+        version,
+        account_contract_address,
+        max_fee,
+        signature,
+        transaction_hash,
+        chain_id,
+        nonce
+    } = tx_info_mock;
+
+    let (is_version_set, version) = is_set_and_value(version, 0);
+    let (is_acc_address_set, account_contract_address) = is_set_and_value(account_contract_address, 0);
+    let (is_max_fee_set, max_fee) = is_set_and_value(max_fee, 0_u128);
+    let (is_tx_hash_set, transaction_hash) = is_set_and_value(transaction_hash, 0);
+    let (is_chain_id_set, chain_id) = is_set_and_value(chain_id, 0);
+    let (is_nonce_set, nonce) = is_set_and_value(nonce, 0);
+    let (is_signature_set, signature) = is_set_and_value(signature, ArrayTrait::new());
```

Comment:
I would just do `serialize_input` (returning a tuple of felts mby) so that it's more explicit about what actually it does